zeppelin-zengine/src/main/java/org/apache/zeppelin/interpreter/InterpreterFactory.java

```diff
@@ -69,24 +69,25 @@
   DependencyResolver depResolver;
 
   public InterpreterFactory(ZeppelinConfiguration conf,
-      AngularObjectRegistryListener angularObjectRegistryListener,
-      DependencyResolver depResolver)
+                            AngularObjectRegistryListener angularObjectRegistryListener,
+                            DependencyResolver depResolver)
```

Comment:
Can you please change indention here and below back, to make sure it is consistent with the rest of the project code?

Here is the relevant style guide section [javaguide.html#s4.6.3-horizontal-alignment](https://google.github.io/styleguide/javaguide.html#s4.6.3-horizontal-alignment)